pkg/validate/security_context.go

```diff
@@ -37,6 +38,27 @@ import (
 
 const (
 	nginxContainerImage string = "nginx"
+	localhost           string = "localhost/"
+	// profile which denies sethostname syscall
+	seccompBlockHostNameProfile = `{
```

Comment:
Why not moving those 2 consts into the Seccomp context. No one else is using it.

Reply:
sure I'll move..